schemas/context/segmentdefinition-expression.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition-expression",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Expression",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Segment expression",
+  "definitions": {
+    "segmentdefinition-expression": {
+      "properties": {
+        "xdm:type": {
+          "title": "Type",
+          "type": "string",
+          "description": "Type of expression. For example PQL",
+          "meta:enum" : [
```

Comment:
These again seem to be Adobe specific? Should this be in an adobe specific extension?